easybuild/easyconfigs/c/Coreutils/Coreutils-8.23-GCC-4.9.2.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'ConfigureMake'
+
+name = "Coreutils"
```

Comment:
out of curiosity: why do you want/need this? This will be installed by default on most distro's?